tests/console/proxyscc_postreg.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2014-2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Do the registration against SCC after installation
+# Maintainer: Yi Xu <yxu@suse.de>
+
+use strict;
+use base "y2logsstep";
+
+use testapi;
+use registration;
+
+sub run {
+    return if get_var("HDD_SCC_REGISTERED");
+    my $version  = get_required_var("VERSION");
+    my $arch     = get_required_var("ARCH");
+    my $reg_code = get_required_var("SCC_REGCODE");
+    my $scc_url  = get_var("SCC_URL");
+
+    select_console 'root-console';
+    assert_script_run "SUSEConnect --url $scc_url" if $scc_url;
```

Comment:
I've tested this in latest builds, not sure how it worked before, but if we redefined $scc_url, we should provide reg code within same command. So, it should be "SUSEConnect --url $scc_url -r $reg_code". Please, change this accordingly, but still having a fallback to registration against official SCC when SCC_URL is not defined.